protos/bottle/notification/organization/v1/created.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package bottle.notification.organization.v1;
+
+import "bottle/account/v1/organization.proto";
+
+message Created {
+  bottle.account.v1.Organizatio organization = 1;
```

Comment:
```suggestion
  bottle.account.v1.Organization organization = 1;
```
We need to make some decisions about how we want to handle events that occur for a resource. Do we make these "Created" ("Updated", etc) messages or do we take a different approach? I'm not married to this.

Reply:
I'm good with the last word being the verb that occurred. We just need to be consistent about it. Case in point `bottle.support.v1.Question` should be `bottle.support.question.v1.Created`. This also brings up the different between stuff being in the `notification` namespace or just their own.

Reply:
`bottle.support.v1.Question` still needs to exist, just like `bottle.account.v1.User` does. Those are the resources.

Under notifications is where the line is a little blurry. These are notifications in our system but they're also events that take place, which reference the User resource.

Do we want to start doing something like:

`bottle.events.v1.Created` and it have any resource in it or perhaps `bottle.events.support.v1.QuestionCreated`?

Reply:
Another good example for us to think about is Organization Membership.

Do we want to have a "OrganizationJoined" message with both an org and user? What about when they're removed from an organization?

Reply:
I'd probably avoid trying to have a single events namespace and shoehorn every kind of event in that. It will inevitably grow. I like `bottle.events.support.v1.QuestionCreated` (or `bottle.support.events.v1.QuestionCreated`) having a `bottle.support.v1.Question` resource attached to it.

For the org example I picture something like:
`bottle.account.v1.Organization`
`bottle.account.v1.User`

```
message bottle.account.events.v1.OrganizationJoined {
  bottle.account.v1.Organization organization = 1;
  bottle.account.v1.User user = 2;
)
```

```
message bottle.account.events.v1.OrganizationLeft {
  bottle.account.v1.Organization organization = 1;
  bottle.account.v1.User user = 2;
)
```
 
If we go this way we can also extend the events a bit more down the line so we can do things like:

```
message bottle.account.events.v1.OrganizationJoined {
  bottle.account.v1.Organization organization = 1;
  bottle.account.v1.User user = 2;

  bottle.account.v1.User author = 3; # Probably could use a better name
  string message = 4; # Something to display on the invite email or something
  int permissions = 5; # If we add permissions or something
)
```